src/extensions/default/bramble/lib/RemoteEvents.js

```diff
@@ -98,6 +98,13 @@ define(function (require, exports, module) {
         var lastKnownEditorFilePath;
         MainViewManager.on("currentFileChange", function(e, file) {
             if(!file) {
+                // This happens when the last file in working set is deleted.
+                lastKnownEditorFilePath = "";
+                var noFile = {
+                    fullPath: "No File",
+                    filename: "No File"
```

Comment:
I think this is good. Question: do we care about l10n here?  This is API level, not user, so we can deal this in Thimble.  But if so, we need to do it, and I bet we are currently just displaying whatever `filename` is.  Also, I think we cache `fullPath` to try and re-open it on the next startup.  We should figure out what to do here.

Reply:
Thanks for the review.
I think l10n work can be done. 
But, I don't understand how to start with the caching issue you mentioned.

Reply:
The Brackets iframe API that runs within Thimble, and listens for this event, is here:

https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L422-L424

When we start the editor, we do some work here to figure out which file to load, checking for that cached path:

https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L524

Probably if you store `null` vs. `"No File"` it should do we want, but I'd have to research a bit more.